inferflow/run_local_inferflow.sh

```
@@ -0,0 +1,117 @@
+KAFKA_PROVIDER="${KAFKA_PROVIDER:-brew}" # brew | docker | none
+
+ensure_java17() {
+  local java_major
+  java_major="$(java -version 2>&1 | awk -F[\".] '/version/ {print $2}')"
+  if [[ "${java_major}" == "1" ]]; then
+    java_major="$(java -version 2>&1 | awk -F[\".] '/version/ {print $3}')"
+  fi
+
+  if [[ -z "${java_major}" || "${java_major}" -lt 17 ]]; then
+    if command -v brew >/dev/null 2>&1 && brew list --versions openjdk@17 >/dev/null 2>&1; then
+      export JAVA_HOME="$(brew --prefix openjdk@17)/libexec/openjdk.jdk/Contents/Home"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)